res/controllers/Numark-Mixtrack-3-scripts.js

```diff
@@ -366,23 +322,22 @@ LED.prototype.flashOn = function(num_ms_on, value, num_ms_off, flashCount,
     this.flashOnceOn(num_ms_on, value);
 
     if (flashCount !== 1) {
-        // flashcount =0 means permanent flash,
-        // flashcount>0 , means temporary flash, first flash already done,
-        // so we don't need this part  if flashcount=1
-        // permanent timer
+        // flashcount == 0 means permanent flash,
+        // flashcount > 0 , means temporary flash, first flash already done,
+        // so we don't need this part  if flashcount == 1
 
-        this.flashTimer = engine.beginTimer(num_ms_on + num_ms_off,
-            function() {
-                myself.flashOnceOn(false);
-            });
+        // permanent timer
+        this.flashTimer = engine.beginTimer(num_ms_on + num_ms_off, function() {
+            myself.flashOnceOn(false);
```

Comment:
is this:

```javascript
engine.beginTimer(num_ms_on + num_ms_off, 'this.flashOnceOn(false)';
```

preferred over:

```javascript
engine.beginTimer(num_ms_on + num_ms_off, this.flashOnceOn.bind(this, false);
```

I get that I don't need to bind for `this` but I also have an argument I'd like to curry. Most examples I've seen use the string method but I feel like the latter is better practice.